Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -24,6 +30,18 @@ - (void)prepareForReuse {
     self.summaryLabel.text = nil;
 }
 
+-(void)awakeFromNib {
+    [self rememberSettingsFromIB];
+}
+
+-(void)rememberSettingsFromIB {
+    NSRange r = NSMakeRange(0, 1);
+    self.summaryLabelAttributesFromIB = [self.summaryLabel.attributedText attributesAtIndex:0 effectiveRange:&r];
+    self.paddingAboveDescriptionFromIB = self.paddingConstraintAboveDescription.constant;
```

Comment:
I'd lay off the test suggestions here. We are still running long on this ticket and just need to get it out the door. We can always do visual tests during our beta fix period.